source/installing_with_npm/index.html.md.erb

```diff
@@ -26,289 +24,16 @@ To use GOV.UK Frontend with NPM you must:
-
-### Recommended solution
-
-Make `/node_modules/govuk-frontend/govuk/assets` available to your project by routing
-requests for your assets folder there.
-
-For example, if your project uses [express.js](https://expressjs.com/), below is
-a code sample you could add to your configuration:
-
-```JS
-app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
-```
-### Alternative solution
-
-Manually copy the images and fonts from `/node_modules/govuk-frontend/govuk/assets` into a public facing directory in your project. Ideally copying the files to your project should be an automated task or part of your build pipeline to ensure that the GOV.UK Frontend assets stay up-to-date.
-
-The default paths used for assets are `assets/images` and `assets/fonts`. **If your asset folders follow this structure, you will not need to complete the following steps.**
-
-To use different asset paths, also complete the below step(s).
-
-1. Set `$govuk-assets-path`, `$govuk-images-path` and `$govuk-fonts-path` in your project Sass file to point to the relevant directories in your project (this will override the defaults set in `/node_modules/govuk-frontend/govuk/settings/_assets.scss`). Make sure you do this in Sass before importing `govuk-frontend` into your project - see [Importing styles](#importing-styles).
-
-    Example 1:
-
-    ``` SCSS
-    // Include images from /application/assets/images and fonts from /application/assets/fonts
-    $govuk-assets-path: '/application/assets';
-
-    @import "govuk-frontend/govuk/all";
-    ```
-
-    Example 2:
-
-    ``` SCSS
-    // Include images from /images/govuk-frontend and fonts from /fonts
-    $govuk-images-path: "/images/govuk-frontend/";
-    $govuk-fonts-path: "/fonts/";
-
-    @import "govuk-frontend/govuk/all";
-    ```
-
-2. Optional: You can also override the helpers used to generate the asset urls, for example if you are using sass-rails' asset-pipeline functionality. You can do this by setting `$govuk-image-url-function` to the name of the function(s) you wish to use. See `src/govuk/settings/_assets.scss` for more information and examples.
-
-## Include CSS and JavaScript
-
-Add the CSS and JavaScript code to your HTML template:
-
-```html
-<!DOCTYPE html>
-  <head>
-    <title>Example</title>
-    <link rel="stylesheet" href="assets/application.css">
-  </head>
-  <body>
-    <!-- Copy and paste component HTML-->
-    <button class="govuk-button">This is a button component</button>
-    <script src="assets/application.js"></script>
-  </body>
-</html>
-```
+When the installation finishes, the `govuk-frontend` package will be in your `node_modules` folder.
 
-If your service supports Internet Explorer 8, you will need to [generate and
-include a separate stylesheet](/supporting_ie8#writing-styles-that-target-ie8) as well.
+You should now get started by [getting the CSS, assets and JavaScript working](/get-started) with one GOV.UK Frontend component.
```

Comment:
_Super_ minor, but this'll make a request to `/get-started` which will immediately redirect to a `/get-started/` with a trailing slash. From a performance point of view, it's cleaner to add the trailing slash as it saves a request!

```suggestion
You should now get started by [getting the CSS, assets and JavaScript working](/get-started/) with one GOV.UK Frontend component.
```

Reply:
Oh wow, I did not know this! Makes me wonder if we've been serving 301s on Pay docs for years. Thanks for letting me know - will fix across the docs.

Reply:
Yep, it's a minor thing that's not worth worrying about too much, but good to build habits. Note also that e.g. GOV.UK is the exact opposite, and redirects to _remove_ the trailing slash!

I only noticed because the Chrome extension I used to check links highlights links that redirect, which makes it easier to spot.